packages/itwinui-react/src/core/Input/Input.tsx

```diff
@@ -37,7 +37,7 @@ export const Input = React.forwardRef((props, ref) => {
   return (
     <Box
       as='input'
-      className={cx('iui-input', className)}
+      className={cx('iui-input iui-field', className)}
```

Comment:
nit: these should be handled using commas instead of spaces

Reply:
Thought I fixed all of these, must have missed this one.  Fixed.